cypress/automated-tests/youth-pass/personal-contact-sections-required-fields.cy.js

```diff
@@ -38,6 +38,10 @@ describe('Youth Pass Personal and Contact Info Section Required Fields', () => {
         cy.get('#form-element-wrapper_116').within(() => {
             cy.get('div.required-text').should('not.be.visible');
         });
+        // TODO: Remove lines 42-44 when program year choice field is removed from SimpliGov
+        cy.get('#form-element-wrapper_260').within(() => {
```

Comment:
suggestion(conditional): If it is possible to use conditional logic, I think I might suggest simply removing any of these tests saying the program year choice should be visible.